projects/arlas-components/assets/i18n/en.json

```diff
@@ -8,27 +8,25 @@
 	"Zoom to result": "Zoom to result",
 	"Cancel": "Cancel",
 	"Import": "Import",
-	"Fill": "Fill",
+	"Download the layer's data": "Download the layer's data",
+	"CSV": "CSV",
+	"Fill": "Polygon",
 	"Contour": "Contour",
-	"Geo-queries & basemaps": "Geo-queries & basemaps",
 	"Geo-queries": "Geo-queries",
-	"operation": "operation",
-	"a_operation": "operation",
-	"b_geometryPath": "geometry",
 	"of_collection": "of",
 	"geo_query_collection": "collection",
-	"Basemaps": "Basemaps",
-	"Choose one of the following basemaps": "Choose one of the following basemaps",
 	"Apply geo-queries": "Apply",
 	"Set geo-queries": "Set",
 	"Manage layers": "Manage map layers",
 	"show_layers": "Show/hide these layers",
+	"include": "include",
+	"exclude": "exclude",
 	"Attachments": "Attachments",
-	"ASC": "Ascendant",
-	"DESC": "Descendant",
+	"ASC": "Ascending",
+	"DESC": "Descending",
 	"Sort on": "Sort on",
 	"Layers and style": "Layers and style",
-	"Informations": "Informationmerged",
+	"Informations": "Informations",
```

Comment:
I think it's one of the words that doesn't accept a plural form

Reply:
yes, but it does not accept a 'merged' form either

Reply:
I agree